### PR TITLE
BDD: fixed various failures

### PR DESCRIPTION
**Pull request by bdunogier**
See the commit messages for what was fixed in each case.

### CODE REVIEW

**Comment by bdunogier** [comment]
It fixes the scenario that was targetted. Other failures are unrelated.

File: Features/Context/SubContext/CommonActions.php [hunk 1]
```diff
@@ -293,7 +293,14 @@ public function iSeeElementFile($element, $file)
...
-        $this->assertSession()->elementExists('css', 'div.ez-view-dashboardview');
+        $this->assertSession()->elementExists('css', '.ez-view-dashboardview');
+    }
+
+    /**
+     * @Given /^I am redirected to a location view$/
+     */
+    public function iHaveBeenRedirectedToALocationView()
+    {
+        $this->assertSession()->elementExists('css', '.ez-mainviews .ez-locationview-content');
```
Comment (dpobel) [inline Features/Context/SubContext/CommonActions.php]: `.ez-locationview-content` is more an implementation detail, you should instead look for `.ez-mainviews .ez-view-locationviewview`.

Comment (bdunogier) [inline Features/Context/SubContext/CommonActions.php]: Great, thank you.

**Comment by pcardiga** [comment]
+1

**Comment by dpobel** [comment]
+1 !

**Pull request merged**